models/pangolin/avalanche_c/pangolin_avalanche_c_airdrop_claims.sql

```diff
@@ -0,0 +1,43 @@
+{{
+    config(
+        alias='airdrop_claims',
+        materialized = 'incremental',
+        file_format = 'delta',
+        tags=['static'],
+        unique_key = ['recipient', 'tx_hash', 'evt_index'],
+        post_hook='{{ expose_spells(\'["avalanche_c"]\',
+                                "project",
+                                "pangolin",
+                                \'["hildobby"]\') }}'
+    )
+}}
+
+WITH early_price AS (
+    SELECT MIN(minute) AS minute
+    , MIN_BY(price, minute) AS price
+    FROM {{ source('prices', 'usd') }}
```

Comment:
out of curiosity, why does this one use `prices.usd` while velodrome uses `dex.prices`? do some blockchains lack tokens in prices?

Reply:
Oh, `prices.usd` lacks prices for a huge portion of tokens, I checked all of them on coinpaprika and if it isn't there, it's because I couldn't find it, I added the missing ones I could find here #3058

I would love for other price feeds to start being available on dune :)